etc/profile-m-z/meson.profile

```diff
@@ -0,0 +1,16 @@
+# Firejail profile for meson
+# Description: A high productivity build system
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include meson.local
+# Persistent global definitions
+include globals.local
+
+# Allow python3 (blacklisted by disable-interpreters.inc)
+include allow-python3.inc
+
+private-bin meson,python3*
```

Comment:
Yes, it does. If you need other programs, `cutom_target`s, ... just add `ignore private-bin`.

smartass OT comment: meson calls them function AFAIK.

Reply:
> smartass OT comment: meson calls them function AFAIK.

And cmake calls them "commands", but since they're all autoconf-like tools and
since these things are the same as autoconf macros in purpose and function,
it's easier to just call everything macros :p